tests/integration/customizations/s3/test_plugin.py

```diff
@@ -1255,6 +1255,17 @@ def test_normal_output_only_show_errors(self):
         # Check that nothing was printed to stdout.
         self.assertEqual('', p.stdout)
 
+    def test_normal_output_no_progress(self):
+        bucket_name = _SHARED_BUCKET
+        foo_txt = self.files.create_file('foo.txt', 'foo contents')
+
+        # Copy file into bucket.
+        p = aws('s3 cp %s s3://%s/ --no-progress' % (foo_txt, bucket_name))
+        self.assertEqual(p.rc, 0)
+        # Ensure success message was printed
```

Comment:
It might be good to assert there was no progress statement as well in the standard output.

Reply:
Updated.